tests/php/ORM/DataQueryTest.php

```diff
@@ -498,6 +500,99 @@ public function testConditionsIncludeTables()
         $this->assertEmpty(array_shift($arrayResult));
     }
 
+    public static function provideFinalisedQueryResolvesSortColumns(): array
+    {
+        return [
+            'missing field' => [
+                'selectField' => '',
+                'sortField' => 'Title',
+                'expectedSQL' => 'SELECT DISTINCT "DataQueryTest_C"."Title" FROM "DataQueryTest_C" ORDER BY "DataQueryTest_C"."Title" ASC',
+            ],
+            'missing field (fully qualified sort)' => [
+                'selectField' => '',
+                'sortField' => '"DataQueryTest_C"."Title"',
+                'expectedSQL' => 'SELECT DISTINCT "DataQueryTest_C"."Title" AS "_SortColumn0" FROM "DataQueryTest_C" ORDER BY "_SortColumn0" ASC',
+            ],
```

Comment:
Theoretically these should probably behave the same as one another... but this is existing behaviour unrelated to the bug I'm fixing so evaluating (let alone changing) this isn't in scope.